### PR TITLE
I18n fixes

### PR DESCRIPTION
**Pull request by bjornjohansen**
- Update POT file to reflect the changes made in the plugin header
- Change the textdomain to the plugin slug.
- Proper loading of the l18n file.
- Add translations to Norwegian Bokmål (nb_NO)

### CODE REVIEW

**Comment by brunohulk** [comment]
Great!!!

**Comment by philipjohn** [comment]
The plugin slug is actually `fb-instant-articles`, [as used on WordPress.org](https://wordpress.org/plugins/fb-instant-articles/) - let's fix that and get this in :)

File: languages/facebook-instant-articles-wp.pot [hunk 1]
```diff
@@ -33,9 +41,9 @@ msgid "Add support for Instant Articles for Facebook to your WordPress site."
 msgstr ""
 
 #. Author of the plugin/theme
-msgid "Dekode"
+msgid "Automattic, Dekode"
```
Comment (diegoquinteiro) [inline languages/facebook-instant-articles-wp.pot]: Please add Facebook too.

**Comment by philipjohn** [comment]
Punting from 3.0 pending changes

**Comment by everton-rosario** [comment]
This wasn't merged in v3.0.0 @philipjohn .
Needs rebase and to add Facebook as contributor.

**Comment by GaryJones** [comment]
Thanks for this.

The text-domain should match the plugin-slug: `fb-instant-articles`, and all instances of the text domain arg should be updated.

Locale submissions should now be handled via translate.wordpress.org.

**Pull request closed without merging**